### PR TITLE
Use XDG_CONFIG_HOME to find the lem configuration directory.

### PR DESCRIPTION
**Pull request by timmydo**
#434

### CODE REVIEW

File: src/config.lisp [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 (defun lem-home ()
   (or (uiop:getenv "LEM_HOME")
-      (merge-pathnames ".lem/" (user-homedir-pathname))))
+      (uiop:xdg-config-home "lem/")))
```
Comment (seanfarley) [inline src/config.lisp:5]: This should fall back to using `~/.lem` so that current users aren't caught off-guard, right?

**Comment by Sasanidas** [comment]
Maybe we can support both? Like in GNU Emacs https://www.gnu.org/software/emacs/manual/html_node/emacs/Find-Init.html

File: src/config.lisp (hunk 1 above)
Comment (timmydo) [reply src/config.lisp]: I changed the logic to take .lem if it exists first and fallback to xdg-config + 'lem/' if that doesn't already exist. Thoughts?

**Comment by cxxxr** [comment]
Thank you!

**Pull request merged**